### PR TITLE
Displaying images with fresco library in android

### LINKED ISSUE #3163: Displaying Images With Fresco Library In Android
### Introduction paragraph (2-3 paragraphs):
Fresco is a powerful system for displaying images in Android applications.

Fresco takes care of image loading and displaying, so you don't have to worry about the logic on how the image is loaded. It will load images from:
- Network 
- Local storage
- Local resources
and display a placeholder until the image has arrived. It has two levels of cache; one in memory and another in internal storage.
In this article we will see how to setup Fresco in an android application. 

### Key takeaways: 
- What is Fresco
- How is Fresco different from other image processing libraries (Glide and Picasso)
- Unique features of Fresco
- How to setup fresco in android

### References: 
- https://frescolib.org/

### PR DESCRIPTION
**Pull request by brandy-kay**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #3163

### CODE REVIEW

**Comment by Ericgacoki** [comment]
Greetings @brandy-kay,
Welcome to Section's Eng-Ed program. We're excited to have you on board. Thank you for submitting your `first article` for review.
I'll be reviewing it to check and help you fix any errors before being published! :rocket:

File: content/articles/displaying-images-with-fresco-library-in-android/index.md [hunk 1]
```diff
@@ -0,0 +1,1548 @@
...
+
+Check out the entire project on [GitHub](https://github.com/brandy-kay/FrescoLibraryDemo).
+
+### Conclusion
+Fresco is a unique library that offers features such as animations, streaming, image loading, and transformation. Explore more on the other features provided by Fresco from their official documentation to help you in your android development process.
+
+Keep Learning.
+
+### References
+- [Fresco Official Documentation](https://frescolib.org/).Fresco was created by Facebook engineers to make efficient use of memory and improves productivity. Fresco was open-sourced for the first time in early 2015. This library is used by firms including Wikipedia, Twitter, and Redfin for their Android apps, Facebook included.
```
Comment (Ericgacoki) [inline content/articles/displaying-images-with-fresco-library-in-android/index.md:222]: I think the article ends at this point.

File: content/articles/displaying-images-with-fresco-library-in-android/index.md [hunk 2]
```diff
@@ -0,0 +1,1548 @@
...
+Fresco is a logical build-in system for displaying images in mobile applications.
+
+It covers up the issues of image loading and displaying, reducing the tedious work to worry about the logic used to load the image from:
+- Internet
+- Internal storage and resources and present a placeholder until the image arrives.
+
+So let's talk more about Fresco in this article.
+
+### Prerequisites
+- Have Android Studio installed.
```
Comment (Ericgacoki) [inline content/articles/displaying-images-with-fresco-library-in-android/index.md:234]: Everything beyond this point is a duplicate.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]
Hello @brandy-kay, on a quick glance, your article structure looks great - kudos for that :rocket:. However, you might have accidentally duplicated the whole article seven times - for instance, there are seven conclusions 😟.
Please fix this before we proceed.

**Comment by brandy-kay** [comment]
Hello @Ericgacoki I have fixed the duplication error, sorry for the inconvenience

**Review by Ericgacoki — APPROVED** [review]

**Comment by hectorkambow** [comment]
@brandy-kay 

[Displaying images with fresco library in android #3438.pdf](https://github.com/section-engineering-education/engineering-education/files/7027948/Displaying.images.with.fresco.library.in.android.3438.pdf)


👋 Good afternoon - and welcome to the EngEd program.
I'll be running a final review on your PR when ready.
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on plagiarism and what is considered as such.

**Comment by brandy-kay** [comment]
hello @Ericgacoki I have fixed the plagiarism problem

**Comment by Ericgacoki** [comment]
@hectorkambow, please run a second plagiarism check 🙂

**Comment by WanjaMIKE** [comment]
@brandy-kay could you fix the minor plagiarism highlighted in the attached report.
[Displaying images with fresco library in android #3438.pdf](https://github.com/section-engineering-education/engineering-education/files/7074704/Displaying.images.with.fresco.library.in.android.3438.pdf)
 the following report.

**Comment by Ericgacoki** [comment]
Hello @brandy-kay, kindly take a look at the [changes](https://github.com/section-engineering-education/engineering-education/pull/3438/commits/e684d5b4ccd651b3bc3071c4337a6a0a1d7fee30) I've made.

File: content/articles/displaying-images-with-fresco-library-in-android/index.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+```xml
+fresco:progressBarImage="@drawable/progress_bar"
+```
+
+When you run the app, you should expect to see the following output:
+
+![demo](/engineering-education/displaying-images-with-fresco-library-in-android/demo.gif)
+
+### Conclusion
+Fresco is a unique library that offers features such as animations, streaming, image loading, and transformation. Explore more on the other features provided by Fresco from their official documentation to help you in your android development process.
```
Comment (Ericgacoki) [inline content/articles/displaying-images-with-fresco-library-in-android/index.md:216]: Referring to the documentation for "other features" means that you didn't cover them in this article. Please give a none-contradicting conclusion.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]

**Comment by brandy-kay** [comment]
hello @Ericgacoki have updated the conclusion

**Comment by Ericgacoki** [comment]
Looks good :rocket:

**Review by Ericgacoki — APPROVED** [review]

**Comment by ahmadmardeni1** [comment]
Welcome to Section EngEd and congrats on your first article🎉

Please go over the changes I made.
Thank you for the review. @Ericgacoki

**Comment by Ericgacoki** [comment]
Thank you Ahmad.

**Pull request merged**